### ISSUE TITLE
Building solution in docker linux image

### ISSUE THREAD

**Issue by fredmabire** [ArxOne/MrAdvice#147]
Hi,

Thanks for your code, it's very useful and a cool free alternative of Postsharp.

I encountered a problem (like when i try with Postsharp) when i try to build a solution which reference MrAdvice in a linux based docker image.

I get the next build error

`/root/.nuget/packages/mradvice/2.8.5/build/MrAdvice.targets(10,5): error MSB4062: The "MrAdviceTask" task could not be loaded from the assembly /root/.nuget/packages/mradvice/2.8.5/build/../build/MrAdvice.Weaver.exe. Could not load file 
or assembly 'Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. [/app/CrossCutterN_Test/CrossCutterN_Test.csproj]

/root/.nuget/packages/mradvice/2.8.5/build/MrAdvice.targets(10,5): error MSB4062:  Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [/app/CrossCutterN_Test/CrossCutterN_Test.csproj]`

Have you ever try to build a solution on a linux environment ?

Thanks for your reply.

**Comment by picrap** [comment]
Hi, I guess you build using .NET Core (`dotnet build`)? If yes, it's related to #133 .

**Comment by picrap** [comment]
Anyone here?

**Comment by fredmabire** [comment]
I'm here but i don't know how to implement things related to #133 
I still need this work on .net core

**Comment by picrap** [comment]
(Salut le français 😉) However do you use a `dotnet build` or VS make? Currently it does not work at all with `dotnet build `.

**Comment by fredmabire** [comment]
(😁 désolé pour mon anglais...) I use dotnet build. I use docker for continuous integration so when I tried to build the solution in docker container I get an error.
When I build the solution in visual studio I have no problem because it use ms build. 
If there is no implementation for. Net core or standard yet, I agree for contribute but I don't know where I can start.
I tried to download your git project but I can't build it on my computer, i don't know why.
